client/components/mma/switch/SwitchComplete.tsx

```diff
@@ -56,6 +86,79 @@ const extrasStyling = css`
 	}
 `;
 
+const whatHappensNextTextCss = css`
+	width: 100%;
+	margin-left: 0.5rem;
+
+	p {
+		${textSans.medium()}
+		margin-top: 0;
+		margin-bottom: ${space[2]}px;
+	}
+`;
+
+const WhatHappensNext = (props: {
+	currency: string;
+	amountPayableToday: number;
+	email: string;
+}) => {
+	return (
+		<Stack
```

Comment:
Should this has a `space` prop? (Maybe instead of the `margin-top` on the `div` below the heading?)

I'd get rid of the `cssOverrides` and move those styles onto the `div` below too as it's better to avoid adding overrides to Source components unless we really need to.